tests/Woketo/Rfc6455/MessageProcessorTest.php

```diff
@@ -297,4 +297,43 @@ public function testItProcessesOnePingBetweenTwoFragmentedTextMessages()
         $this->assertSame(count($messages[0]->getFrames()), 1);
         $this->assertSame(count($messages[1]->getFrames()), 2);
     }
+
+    public function testItCatchesWrongContinutionFrameException()
+    {
+        $framefactory = $this->prophesize(FrameFactory::class);
+        $framefactory
+            ->createCloseFrame(Frame::CLOSE_PROTOCOL_ERROR)
+            ->willReturn(new Frame(BitManipulation::hexArrayToString(['88','02','03','ef'])))
+            ->shouldBeCalled();
+        $processor = new MessageProcessor($framefactory->reveal());
+
+        $messages = iterator_to_array($processor->onData(
+            BitManipulation::hexArrayToString(['80','98','53','3d','b9','b3','3d','52','d7','9e','30','52','d7','c7','3a','53','cc','d2','27','54','d6','dd','73','4d','d8','ca','3f','52','d8','d7']),
```

Comment:
Please add some comments about data. And notice that you can avoid the array in `hexArrayToString` method